### ISSUE TITLE
What is the state of the project ? 

### ISSUE THREAD

**Issue by sucrecacao** [xi-editor/xi-editor#1298]
Hi !

I'm confuse about the state of the project

I saw the video of Raph Levien at Recurse Center , I though the project was awesome. Then I found Raph's the postmortem article announcing the end of the project. And now I see that people still contribute to the GitHub repo.

As a newbie, I couldn't understand Raph article. I think it was addressing xi's community and contributors. 

Did the project stop because it reached a dead end or did it stop because it didn't reach some expectation ? Do xi still have a future ? If yes what would it be ? Is xi current state a viable backend option to build a text editor or is this just an experiment ?

Thanks

**Comment by jansol** [comment]
TL;DR: as a standalone editor the project has effectively ended, but the components and research are kept alive and well by an active community


As a text editor I'd say the project is in "maintenance mode" where it gets occasional fixes but no active feature development. However the project has sparked a fairly active community over on [zulip](https://xi.zulipchat.com/) as well as a number of crates for text editing, vector rendering, cross-platform GUI and related tasks, all of which are being used and actively developed by said community.

Active development on a full text editor stopped mostly because the main developers shifted focus to cross-platform GUI (started from xi-win and later turned into druid). The GUI work got (AFAIK) funded as part of the [runebender](https://github.com/linebender/runebender) so naturally incentives for prioritizing work on that are higher.

Experimenting with rope structures and asynchronous operations as means to scale to arbitrarily large file sizes was a major goal of the text editor project and has been largely successful. Another goal was to build xi-editor as sort of a guide for text editor developers, so the theory and findings have been [documented](https://xi-editor.io/docs.html) in detail.

The xi-editor core and related crates are perfectly usable for building a text editor as evidenced by [xi-mac](https://github.com/xi-editor/xi-mac) and [Tau](https://gitlab.gnome.org/World/Tau). (other frontends might be in similarly usable shape as well, but I have only tried these two)
